### ISSUE TITLE
ShaderCache duplicate shader overrides ShaderFixes

### ISSUE THREAD

**Issue by bo3b** [bo3b/3Dmigoto#12]
This looks like it's the other way around, but DHR has reported that he got no results from ShaderFixes changes until he deleted the dup shader.

**Comment by DarkStarSword** [comment]
Does this issue still exist? I noticed some of the shader preload code looked suspect as I was passing through, but haven't looked at it in detail.

**Comment by bo3b** [comment]
Ah, no, this is fixed.  At some point I disabled the ShaderCache from being to override anything.  It's just the dump cache now, not ever loaded live.  

I did that to make sure that unwanted Decompiler problems didn't damage any game items, and also to simplify the use case and make it clear that *only* ShaderFixes get loaded.  Too complicated to explain and use ShaderCache, and not enough value.

 Not positive I caught the pre-load case, can't recall.  Chiri has said that it fixed problems to preload them, but I think we haven't seen those problems or so many code changes have altered it.  I don't know of any active game fixes that use the feature.  My medium preference would be to remove the pre-load feature altogether as it adds a lot of bizarre complexity.
